applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -871,7 +1005,11 @@ static void usb_init_next_status_cb(struct usbd_contex *const contex,
 	}
 
 	if (new_state != state) {
-		broadcast_usb_state(new_state);
+		if ((state == USB_STATE_SUSPENDED) && is_usb_active_next()) {
+			broadcast_usb_state(USB_STATE_ACTIVE);
+		} else {
+			broadcast_usb_state(new_state);
+		}
```

Comment:
Since `USB_STATE_ACTIVE` support was introduced together with adding USB HID support, I think we could keep it as is. The changes go in together anyway.